nemo/collections/nlp/modules/common/megatron/module.py

```diff
@@ -15,10 +15,21 @@
 """Megatron Module"""
 
 import torch
-from apex.transformer import parallel_state, tensor_parallel
 from torch.autograd import Variable
 from torch.nn.parameter import Parameter
 
+try:
```

Comment:
@ericharper also in https://github.com/NVIDIA/NeMo/blob/r1.6.0/nemo/collections/nlp/modules/common/megatron/megatron_bert.py#L20

Reply:
Was actually able to remove that file all together.